src/lib/y2partitioner/dialogs/popup.rb

```diff
@@ -41,7 +40,8 @@ def layout
           1, # left / right
           0.45, # top / bottom
           VBox(
-            Id(:help_text_container),
+            # This Id makes the Help texts work, see Wizard.SetHelpText
```

Comment:
NP: Maybe we can explain a bit more here, what do you think?

```suggestion
            # This Id makes the Help texts work because CWM will assign the merged help texts
            # of the individual widgets to that one, see Wizard.SetHelpText
```